src/game_battlealgorithm.cpp

```diff
@@ -252,20 +252,23 @@ void Game_BattleAlgorithm::AlgorithmBase::PlayAnimation(bool on_original_targets
 
 	Game_Battle::ShowBattleAnimation(
 		GetAnimation()->ID,
-		anim_targets);
+		anim_targets,
+		false,
+		-1,
+		invert);
```

Comment:
Doesn't have to be done in this PR, but when you get this level of mess it's time to convert these defaulted parameters to a `BattleAnimationArgs` struct.